applications/CableNetApplication/custom_elements/sliding_cable_element_3D.hpp

```diff
@@ -194,6 +194,9 @@ namespace Kratos
 
     private:
 
+        // boolean for the cable --> does not resist to compression
+        bool mIsCompressed;

```

Comment:
Huum, I would use a flag

Reply:
this is consistent with the cable element. We can change this at some point altogether